### PR TITLE
Feature/front page layout

### PR DESCRIPTION
**Pull request by sharph**

### CODE REVIEW

File: ckanext/odp_theme/plugin.py [hunk 1]
```diff
@@ -31,4 +43,6 @@ def update_config(self, config):
     def get_helpers(self):
         """Register most_recent_datasets function"""
```
Comment (ddohler) [inline ckanext/odp_theme/plugin.py]: This docstring needs to be updated.

File: ckanext/odp_theme/plugin.py [hunk 2]
```diff
@@ -31,4 +43,6 @@ def update_config(self, config):
...
 
-        return {'odp_theme_most_recent_datasets': most_recent_datasets}
+        return {'odp_theme_most_recent_datasets': most_recent_datasets,
+                'odp_theme_dataset_count': dataset_count,
+                'odp_theme_groups': groups}
```
Comment (ddohler) [inline ckanext/odp_theme/plugin.py:48]: You could potentially save a few lines by rewriting `dataset_count` and `groups` as lambdas; they're pretty short. But it's not a big deal--this way is good too.

**Comment by ddohler** [comment]
+1 -- do you happen to know how the `h` context variable gets set in the template?

**Comment by sharph** [comment]
I assume it gets built as all the plugins and and features are loaded and is then passed to the template renderer upon render, but I haven't seen the internals.

**Pull request merged**